backend/models/Bounty.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const BountySchema = new Schema({
+    title: {
+        type: String,
+        required: true
+    },
+    description: {
+        type: String,
+        required: true
+    },
+    reward: {
+        type: String,
+        required: true
+    },
+    status: {
+        type: Boolean,
+        default: false
+    },
+    createdBy: {
+        type: String
+    },
+    claimedBy: {
```

Comment:
This likely needs to be a "join" field where you are relating it to another collection, correct?

Reply:
https://stackoverflow.com/a/18002078

Reply:
Thank you for the resource! I'm going to add it once we get a final set up of users. It's firebase users so I'm not sure how that translates, but I'm sure it won't be too different.

Reply:
Ah yes potentially different setup for firebase users.